CHANGELOG.md

```diff
@@ -1,7 +1,24 @@
-## 0.x.x - Unreleased
+## 0.4.0 - 20 November 2018
 
 ### Added
+* Added support for self-signed certificates and reading from Windows/Mac certificate stores (currently opt-in) [#613](https://github.com/Microsoft/vscode-docker/issues/613), [#602](https://github.com/Microsoft/vscode-docker/issues/602), [#483](https://github.com/Microsoft/vscode-docker/issues/483)
+* Use a different icon for unhealthy containers (thanks @grhm) [#615](https://github.com/Microsoft/vscode-docker/issues/615)
+* 8.9-alpine -> 10.13-alpine [#624](https://github.com/Microsoft/vscode-docker/pull/624)
 * Adds preview support for debugging .NET Core web applications running in Linux Docker containers.
+* Azure Container Registry improvements:
+  - Automatic login for pulls (even if Admin user not enabled)
+  - Navigate and build tasks
```

Comment:
What does it mean to "navigate" a task?

Should this be "View and build tasks"?

Reply:
Meant "explore"